blt/src/Blt/Plugin/Commands/UsersCommands.php

```diff
@@ -42,9 +42,15 @@ public function loadUsers() {
         ];
       }
       else {
-        $this->logger->warning("Skipping admin user. You must provide both the username and the password.");
+        $this->logger->warning("Skipping admin user. You must provide both the username, password & enable key value.");
       }
     }
+    else {
+      $commands = [
+        'cgov:acsf:block-admin' => [],
+      ];
+      $this->logger->warning("Admin has been blocked.");
+    }
```

Comment:
There is no check for whether the `admin` user should actually be enabled. At present, this code will enable the user as long as YAML data structure is set, regardless of the value set in the `enabled` key.

The pseudocode should be

```
if the admin key exists {
  if the admin block is valid {
    if the enabled key is present AND set to true {
      enable the admin user
    }
    else {
      disable the admin user.
    }
  }
  else {
    disable the admin user.
    log a warning.
  }
}
else {
  disable the admin user.
}
```

For the messages....

On the warning message, let's tweak the text to say 
"Skipping admin user. You must provide both the username and password and set enable to either true or false."

The other cases are all normal operation and at most should be logged as `debug` messages, but **only** if they won't normally be output.  (Our BLT output is already quite noisy.)